### ISSUE TITLE
the app breaks entirely after modding cors

### ISSUE THREAD

**Issue by rehashedprime** [iamshaunjp/graphql-playlist#9]
is there another method to achieve the same thing perhaps just running react on the same express server

**Comment by ImCatYao** [comment]
Yes, It must be. You can write your own express server and Webpcak config file in the same folder

**Comment by ImCatYao** [comment]
> is there another method to achieve the same thing perhaps just running react on the same express server

I have not meet any problem with cors yet.

**Comment by bngsudheer** [comment]
Can you describe the problem? Do you get any error message?
